src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -30,69 +34,181 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
-            tdprSlotsService.updateSlots(slots, id, startDate, endDate).then(function () {
-                $scope.showSubmitSuccess = true;
-                $timeout(function () {
-                    $scope.showSubmitSuccess = false;
-                }, 2000);
-            });
-        };
 
-        $scope.showPreviousWeek = function () {
-            $scope.relativeDayNumber -= 7;
-            $scope.clearTable();
-            $scope.getSlots(id);
-        };
+            tdprSlotsService.updateSlots(slots, id, startDate, endDate).then(function(){
+                            $scope.showSubmitSuccess = true;
+                            $timeout(function(){$scope.showSubmitSuccess=false;}, 2000);
+                        });
 
-        $scope.showNextWeek = function () {
-            $scope.relativeDayNumber += 7;
-            $scope.clearTable();
-            $scope.getSlots(id);
-        };
+            var note = {
+                         "id": $scope.noteContent.id,
+                         "person": {
+                           "id": id
+                         },
+                         "description": $scope.temporaryContent,
+                         "date": $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber), "yyyy-MM-dd")
+             }
+
+            sendNote(note);
+            $scope.hasSlotChanged = false;
+            $scope.hasNoteChanged = false;
+         };
+
+        function enableNoteEditing() {
+            $scope.hasNoteChanged = true;
+            $scope.buttonTitle = "Discard";
+            $scope.textAreaStyle = {'background-color':'white'};
+            $scope.editNote = true;
+        }
+
+        function disableNoteEditing() {
+            $scope.hasNoteChanged = false;
+            $scope.buttonTitle = "Edit note";
+            $scope.textAreaStyle = {'background-color':LIGHTGREYCOLOUR};
+            $scope.temporaryContent = $scope.noteContent.description;
+            $scope.editNote = false;
+        }
 
-        $scope.getSlots(id);
+        function sendNote(note) {
+            tdprPersonService.updateNote(note).then(function(response) {
+            }, function(failure) {
+                console.log(failure);
+            });
+        }
 
         $scope.markSlots = function(slot) {
             slot.type === $scope.currentType ? slot.type = AvailabilityEnum.empty.id : slot.type = $scope.currentType
         };
 
+        function getNote(personId, date) {
+            tdprPersonService.getNote(personId, date).then(function(response) {
+                $scope.noteContent = response.data;
+                $scope.temporaryContent = response.data.description;
+                if(response.status == 204) {
+                    $scope.noteContent.id = null;
+                }
+            }, function(failure) {
+                console.log("getNote problem: Status: " + failure.status);
```

Comment:
If there's an error we need to show - show it to the user - otherwise don't log anything